packages/palette/src/elements/BarChart/Bar.tsx

```diff
@@ -185,6 +188,16 @@ export const Bar = ({
         </HighlightLabel>
       )}
       {hover && label && <BarHoverLabel>{label}</BarHoverLabel>}
+      <XAxisLabel color="black60" size="2">
+        {xAxisLabel}
+      </XAxisLabel>
     </BarBox>
   )
 }
+
+const XAxisLabel = styled(Sans)`
+  position: absolute;
+  bottom: -${space(2)}px;
```

Comment:
This part is a bit too much a hack and bothers me. It is because the label is inside the `<BarBox>` to have the horizontal alignment right but then had to do a negative margin bottom here (which is dependent on the height of the label).

Reply:
you could do `top: calc(100% + ${marginBetweenLabelAndBaseLine});`